src/mmw/apps/export/tasks.py

```diff
@@ -46,6 +46,12 @@ def update_resource(user_id, project_id, params):
             'name': md.get('name'),
             'contents': to_gms_file(mdata).read() if mdata else None,
         })
+    for wd in params.get('weather_data', []):
```

Comment:
The implementation for `update` and `create_resource` looks identical. Consider consolidating to a helper function to avoid a drift of implementation on subsequent updates.

Reply:
Thanks for prompting that. Refactored in aac7ce9.